app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt

```diff
@@ -28,15 +26,13 @@ class MultiselectSelectionMenu(
                 context.getString(R.string.tab_tray_multiselect_menu_item_bookmark),
                 textColorResource = R.color.primary_text_normal_theme
             ) {
-                context.components.analytics.metrics.track(Event.TabsTraySaveToCollectionPressed)
```

Comment:
Should we also be removing the telemetry from `Event.kt`?

Reply:
Good point - I ended up reusing the event and moving it to the new button that does the actual collection saving (had to rework some of the metrics and add some kdocs too)